src/mlpack/core/data/dataset_mapper.hpp

```diff
@@ -121,8 +119,8 @@ class DatasetMapper
   template<typename Archive>
   void Serialize(Archive& ar, const unsigned int /* version */)
   {
-    ar & data::CreateNVP(types, "types");
-    ar & data::CreateNVP(maps, "maps");
+    ar& data::CreateNVP(types, "types");
+    ar& data::CreateNVP(maps, "maps");
```

Comment:
I don't know about this in detail but is it not pass by reference? Because clang-format simply treats it as pass by reference and aligns `&` with the datatype returned.